src/dll/edbgEcmdDll.C

```diff
@@ -1076,6 +1084,18 @@ uint32_t addChipUnits(const ecmdChipTarget &i_target,
     rc = p10x_convertPDBGClassString_to_CUString(class_name, cuString);
     pdbg_for_each_target(class_name.c_str(), i_pTarget, target) {
 
+      // If it is a chipunit, it could be under ocmb. So ignore it is
```

Comment:
ignore if it is not under ocmb

Reply:
This adds pu targets. So if the chipunit is under ocmb, it will skip.  (not of npos means ocmb found) 

if (targetPath.find(substring) != std::string::npos) { continue; }: This is an if statement that checks if the substring "ocmb" is present in the targetPath string using the find method. If the substring is found (!= std::string::npos), the continue statement is executed, which means the code skips the rest of the loop's body and proceeds to the next iteration.